content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -145,25 +153,27 @@ cases precedence will be given first to the longest matching path. If two paths
 are still equally matched, precedence will be given to paths with an exact path
 type over prefix path type.
 
+## Hostname Wildcards
+Hosts can be precise matches (for example “`foo.bar.com`”) or a wildcard (for example “`*.foo.com`”).
+Precise matches require that the http host header matches the Host setting.
+Wildcard matches require the http host header is equal to the suffix of the wildcard rule.
+
+| Host        | Host header       | Match?                                            |
```

Comment:
Ideally: add a caption for the table. See https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#table-captions for details.

Reply:
(it's OK to fix that after this merges, or to leave that detail not done)

Reply:
I was unable to get this table formatting to work. Can you point to an example where we do this within the current docs? For me, it kept messing up the format of the table, but it might be that this caption requires some additional formatting beyond the norm.